doc/source/whatsnew/v0.25.0.rst

```diff
@@ -64,6 +64,42 @@ is respected in indexing. (:issue:`24076`, :issue:`16785`)
     df = pd.DataFrame([0], index=pd.DatetimeIndex(['2019-01-01'], tz='US/Pacific'))
     df['2019-01-01 12:00:00+04:00':'2019-01-01 13:00:00+04:00']
 
+Concatenating Sparse Values
```

Comment:
add a ref

Reply:
I have seen you make this review comment before, but IMO, there is not really much added value in asking for this. It doesn't change anything in the built documentation, and is only needed if you make a link to it from somewhere else in our docs (and in this case, I think it is rather unlikely that we will ever link to it).

Reply:
disagree. as a matter of course if we have subsections we should have links. its not much of a burden and promotes consistency.

Reply:
It's just that you are asking something (and we are often already asking a lot of contributors in many rounds) that is basically useless, except for the consistency argument that you give (IMO that's not worth the overhead of always adding it, but OK, we can disagree on that).

> as a matter of course if we have subsections we should have links

Note that this does not introduce *links*. Things like anchors so that people can link to that section on the html docs, that is done automatically by sphinx, you don't need this label for that.

Reply:
again disagree. consistency is SO important in the pandas docs / code. If it can be automated (either the actual code / docs or the check of it great), otherwise the purpose is NOT to special case the world. Again having to make that judgement call on each PR would make a big inconsistency across PR's & maintainers.

Reply:
> having to make that judgement call on each PR would make a big inconsistency across PR's & maintainers.

The thing is that you don't need to make a judgement call here: it simply is not needed for new sections being added. Only when you ask to add somewhere an internal link, you need to add a label.

Reply:
you are still missing the point

pls add the ref here - arguing about reviews is a waste of time